src/DetailsCard/DetailsCard.js

```diff
@@ -1,14 +1,13 @@
 import './DetailsCard.css';
 
 const DetailsCard = ({ details }) => {
-  console.log('details card', details)
-
   if (!details) {
```

Comment:
What is this code snippet doing here? Do we need it still? <3

Reply:
Thanks. Great catch!